inventory.py

```diff
@@ -34,16 +40,23 @@
 SUBSCRIPTION_NAME = ENV['API_SUBSCRIPTION_NAME']
 API_SCOPE_PREFIX = ENV['API_SCOPE_PREFIX']
 MAX_REQ_ATTEMPTS = ENV['MAX_REQ_ATTEMPTS']
+CANVAS_TOKEN = ENV['CANVAS_TOKEN']
+CANVAS_URL = ENV['CANVAS_URL']
+PUBLISHED_DATE_WORKERS = ENV.get('PUBLISHED_DATE_WORKERS',8)
```

Comment:
Could this be generalized in case we end up using asynchronous calls for something else? Maybe `ASYNC_WORKERS`?

Reply:
```suggestion
PUBLISHED_DATE_WORKERS = ENV.get('PUBLISHED_DATE_WORKERS', 8)
```

Reply:
Or maybe `NUM_ASYNC_WORKERS` would be more clear?

Reply:
Done